src/data/full-quiz.js

```diff
@@ -1,4 +1,59 @@
 const fullQuiz = [
+  {
+    Question:
+      "Which method returns an HTMLCollection object that consists of all of the elements that have the same class name?",
+    Answer: "getElementsByClassName()",
+    Distractor1: "querySelector()",
+    Distractor2: "getElementsByName()",
+    Distractor3: "getElementsByTagName()",
+    Explanation:
+    "The getElementsByClassName method of Document interface returns an array-like object of all child elements which have all of the given class name(s).",

```

Comment:
I think the explanation can be simplified.

Proposed fix: The getElementsByClassName method returns an object containing all of the child elements with that given class name. 

I think simpler is always better. And if people want to learn more about this method they can always click on the MDN article for the official definition.

Reply:
Sounds good. Thank you for the feedback. :-)